controllers/admin/AdminModulesController.php

```diff
@@ -1383,7 +1383,7 @@ public function initModal()
         parent::initModal();
 
         $this->context->smarty->assign(array(
-            'trad_link' => 'index.php?tab=AdminTranslations&token='.Tools::getAdminTokenLite('AdminTranslations').'&type=modules&lang=',
+            'trad_link' => 'index.php?tab=AdminTranslations&token='.Tools::getAdminTokenLite('AdminTranslations').'&type=modules&module='.$_REQUEST['configure'].'&lang=',
```

Comment:
yeah but it's available in $_POST, right? so you could use `Tools::getValue` just to keep standards